### PR TITLE
dump.Parse handle string line trim last \n or \r\n

### PR DESCRIPTION
**Pull request by Jasonzz-he**
on windows someone has \r\n end

### CODE REVIEW

**Comment by Jasonzz-he** [comment]
add strings package

**Comment by Jasonzz-he** [comment]
dump.Parse handle string line trim last \n or \r\n #216
on windows someone has \r\n end

File: dump/parser.go [hunk 1]
```diff
@@ -48,7 +49,7 @@ func Parse(r io.Reader, h ParseHandler, parseBinlogPos bool) error {
 			break
 		}
 
-		line = line[0 : len(line)-1]
+		line = strings.SplitAfter(line, ";")[0]
```
Comment (siddontang) [inline dump/parser.go]: can you add an explanation here?

File: dump/dump.go [hunk 2]
```diff
@@ -123,7 +123,7 @@ func (d *Dumper) Dump(w io.Writer) error {
 	}
 
 	if d.maxAllowedPacket > 0 {
-		args = append(args, fmt.Sprintf("--max_allowed_packet=%dM", d.maxAllowedPacket))
+		args = append(args, fmt.Sprintf("--max-allowed-packet=%dM", d.maxAllowedPacket))
```
Comment (siddontang) [inline dump/dump.go]: please use another PR. One PR one thing :-)

**Pull request closed without merging**